test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -566,9 +566,75 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			By("Saving the old performance profile")
 			initialProfile = profile.DeepCopy()
 		})
+		When("RealTime Workload with RealTime Kernel set to false", func() {
+			It("[test_id:50991][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
+				By("Modifying profile")
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					HighPowerConsumption: pointer.Bool(false),
+					RealTime:             pointer.Bool(true),
+				}
+
+				profile.Spec.RealTimeKernel = &performancev2.RealTimeKernel{
+					Enabled: pointer.BoolPtr(false),
+				}
+
+				By("Updating the performance profile")
+				profiles.UpdateWithRetry(profile)
+
+				By("Applying changes in performance profile and waiting until mcp will start updating")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+
+				By("Waiting for MCP being updated")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+
+				stalldEnabled, rtKernel := true, false
```

Comment:
same here about rtkernel

Reply:
As mentioned above in this test case we want to check the behaviour by explicitly specifying realtime kernel to false